compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.h

```diff
@@ -104,15 +104,41 @@ FailureOr<Flow::DispatchRegionOp> wrapOpInDispatchRegion(RewriterBase &rewriter,
 /// into a dispatch region.
 bool isClonableIntoDispatchOp(Operation *op);
 
-/// Returns true if the operation has dequantization-like properties.
+/// Returns true if the operation increases/decreases bitwidths of tensors.
 /// This function checks that the genericOp:
-///     1. Has only one output, and the output has an identity indexing map
-///     2. Has all parallel loops.
-///     3. Has exactly one input with an identity indexing map.
-///     4. All other inputs are projected permutations and not permutations.
-///     5. The input with an identity indexing map has a smaller element
-///        bitwidth than the output
-bool isDequantizationLikeOp(Operation *op);
+/// 1. Has only one output.
+/// 2. Has all parallel loops.
+/// 3. Compared to the element type of the input with highest rank,
+///    the output element type has either a higher or lower bitwidth.
+struct BitWidthChangeInfo {
+  // Return the operand the recognizer treats as the "input".
+  // Is guaranteed to be a `RankedTensorType`.
+  OpOperand *inputOperand;
```

Comment:
```suggestion
  OpOperand *inputOperand = nullptr;
```

Since this struct doesn't have any constructor to initialize this.